vhdl/generate_vhdl.m

```diff
@@ -0,0 +1,26 @@
+% Kill existing compile
+if bdroot ~= "" && strcmp(get_param(bdroot,'SimulationStatus'), 'stopped') == 0
+    set_param(gcs, 'SimulationCommand', 'stop')
+    cmd = [bdroot,'([],[],[],''term'');'];
+    try
+        eval(cmd)
+    catch
+    end
+end
+if exist('mp','var') == 0
+    sm_run_me_first;
+    %pause(3)
+end
+cmd = [bdroot,'([],[],[],''compile'');'];
+    try
+        eval(cmd)
```

Comment:
Is there an alternative to using the eval function here?  Could we simply call the bdroot command?

Reply:
Yeah, I was definitely thinking there would be but there isn't that I can tell. The reason this is being done this way is the [bdroot](https://www.mathworks.com/help/simulink/slref/bdroot.html) command actually just returns model name as a string, and what is actually being run is the [model](https://www.mathworks.com/help/simulink/slref/model_cmd.html) command which requires the model name as the function name. So above for example it is really running `SE([],[],[],'compile')` when compiling the echo model.  

While SE() is more clear, that would require a small change for each model manually and remove the ability for the `generate_vhdl` file to be in a single location and work every time.

Reply:
I believe Ross uses `eval` as well in some of the simulink processing code.

Reply:
Gotcha.  Just wanted to make sure we weren't using it unnecessarily since it's generally a better practice not to use eval b/c it can make the code difficult to read and can be less efficient.